### ISSUE TITLE
Uncaught TypeError: Cannot read properties of null (reading 'useMemo')

### ISSUE THREAD

**Issue by shan-du** [functional-ui/functional-ui-kit#10]
## The package simply does not work under a few react frameworks

Receiving this error when trying to render any FUI component
```
Uncaught TypeError: Cannot read properties of null (reading 'useMemo')
```

Starter Environments (react 18.2.0, react-dom 18.2.0):
- latest `create-react-app`: https://github.com/facebook/create-react-app
- latest `vite-react-starter`: https://github.com/fabri4c/vite-react-starter

### Steps to Reproduce:
1. Install any of the 2 local dev environments mentioned above
2. Install `functional-ui-kit`
3. Import any `functional-ui-kit` component & add it to main `render` function in `App.js` (see code snippet in the end)
4. Run `npm run start` (for `create-react-app`) OR `npm run dev` (for `vite-react-starter`)
5. Page does not render at all & receives above mentioned error

#### App.js Code
```
import { FuiBadge } from 'functional-ui-kit/fui-badge';
import 'functional-ui-kit/style';

function App() {
  return (
    <div>
      <FuiBadge label="label" />
    </div>
  );
}

export default App;
```

**Comment by AlmogAdziashvili** [comment]
Hi @shan-du, thanks for reporting this issue!
This should be fixed on version 1.0.1, please try and let me know if I can close this issue.

**Comment by shan-du** [comment]
Hey @AlmogAdziashvili it works now!! Thank you for the super quick fix 🙌
